src/EnergyPlus/PurchasedAirManager.cc

```diff
@@ -214,907 +214,908 @@ void GetPurchasedAir(EnergyPlusData &state)
+                            PurchAir.Name);
         SetupOutputVariable(state,
                             "Zone Ideal Loads Heat Recovery Total Heating Rate",
                             Constant::Units::W,
-                            PurchAir(PurchAirNum).HtRecTotHeatRate,
+                            PurchAir.HtRecTotHeatRate,
                             OutputProcessor::TimeStepType::System,
                             OutputProcessor::StoreType::Average,
-                            PurchAir(PurchAirNum).Name);
+                            PurchAir.Name);
         SetupOutputVariable(state,
                             "Zone Ideal Loads Heat Recovery Sensible Cooling Rate",
                             Constant::Units::W,
-                            PurchAir(PurchAirNum).HtRecSenCoolRate,
+                            PurchAir.HtRecSenCoolRate,
                             OutputProcessor::TimeStepType::System,
                             OutputProcessor::StoreType::Average,
-                            PurchAir(PurchAirNum).Name);
+                            PurchAir.Name);
         SetupOutputVariable(state,
                             "Zone Ideal Loads Heat Recovery Latent Cooling Rate",
                             Constant::Units::W,
-                            PurchAir(PurchAirNum).HtRecLatCoolRate,
+                            PurchAir.HtRecLatCoolRate,
                             OutputProcessor::TimeStepType::System,
                             OutputProcessor::StoreType::Average,
-                            PurchAir(PurchAirNum).Name);
+                            PurchAir.Name);
         SetupOutputVariable(state,
                             "Zone Ideal Loads Heat Recovery Total Cooling Rate",
                             Constant::Units::W,
-                            PurchAir(PurchAirNum).HtRecTotCoolRate,
+                            PurchAir.HtRecTotCoolRate,
                             OutputProcessor::TimeStepType::System,
                             OutputProcessor::StoreType::Average,
-                            PurchAir(PurchAirNum).Name);
-
+                            PurchAir.Name);
+        SetupOutputVariable(state,
+                            "Zone Ideal Loads Zone Heating Fuel Energy Rate",
+                            Constant::Units::W,
+                            PurchAir.ZoneTotHeatFuelRate,
+                            OutputProcessor::TimeStepType::System,
+                            OutputProcessor::StoreType::Average,
+                            PurchAir.Name);
+        SetupOutputVariable(state,
+                            "Zone Ideal Loads Zone Cooling Fuel Energy Rate",
+                            Constant::Units::W,
+                            PurchAir.ZoneTotCoolFuelRate,
+                            OutputProcessor::TimeStepType::System,
+                            OutputProcessor::StoreType::Average,
+                            PurchAir.Name);
```

Comment:
Added two energy consumption rate report variables for zones: `Zone Ideal Loads Zone Heating Fuel Energy Rate` and `Zone Ideal Loads Zone Cooling Fuel Energy Rate`.